extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -203,6 +203,7 @@ export function folderAlreadyExists(filename: string) { return localize('folderA
 export function fileAlreadyAddedToProject(filepath: string) { return localize('fileAlreadyAddedToProject', "A file with the path '{0}' has already been added to the project", filepath); }
 export function folderAlreadyAddedToProject(folderpath: string) { return localize('folderAlreadyAddedToProject', "A folder with the path '{0}' has already been added to the project", folderpath); }
 export function invalidInput(input: string) { return localize('invalidInput', "Invalid input: {0}", input); }
+export function invalidProjectPropertyValue(propertyName: string) { return localize('invalidPropertyValue', "Invalid value specified for the property '{0}' in .sqlproj file", propertyName); }
```

Comment:
Let me know if there is anything else that needs to be done for localization. I don't see these values in `package.nls.json`, so I didn't add this new one either.

Reply:
There's two types of ways that things get localized - one is in package.nls.json. This is only for strings in a package.json. The other is through localize calls in the ts code. (In the end they both go through the same package though)

So what you have here is correct, and would be similar to any other localized strings that aren't in the package.json itself. 

(other resources outside of those two places are a whole separate matter and not something supported directly by ADS/VS Code)